client/my-sites/stats/grid-layout.scss

```diff
@@ -129,27 +129,6 @@ $grid-vertical-gutters: 32px;
 		padding: 0 16px;
 	}
 
-	// Temporary change until all cards are replaced with new components. Remove when cleaning 'stats/insights-page-grid' FF.
-	@media (min-width: 661px) {
-		.stats__module-wrapper--tags-categories {
-			display: flex;
-			flex-direction: column;
-			align-items: stretch;
-
-			& > * {
-				margin: 0;
-
-				& + div {
-					flex: 1 0 auto;
-				}
-			}
-
-			.card.stats-module {
-				margin-bottom: 0;
-			}
-		}
-	}
-
 	.stats__module-wrapper--tags-categories,
```

Comment:
```suggestion
```

The class name `stats__module-wrapper--tags-categories` seems to be unused anymore.

Reply:
Great observation @dognose24. There are more classes like this one and I can remove them when working on [#75742](https://github.com/Automattic/wp-calypso/issues/75742). I'll add it as one of the requirements of the other issue.